grammars/ini.cson

```diff
@@ -1,5 +1,6 @@
 'fileTypes': [
   'cfg',
+  'conf',
```

Comment:
See https://github.com/jacobbednarz/atom-language-ini/pull/28#issuecomment-209121923 for reasoning why this hasn't landed before.